.travis.yml

```diff
@@ -1,15 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - stable
-  - 7.6
-matrix:
-  include:
-    - node_js: "8.0"
-      env: "NVM_NODEJS_ORG_MIRROR=https://nodejs.org/download/nightly"
-  allow_failures:
-    # Allow the nightly installs to fail
-    - env: "NVM_NODEJS_ORG_MIRROR=https://nodejs.org/download/nightly"
```

Comment:
i'm okay with testing node 6 as long as it doesn't require work (i.e. babelfying everything)